packages/frontend/app/components/instructor-group/courses.hbs

```diff
@@ -11,7 +11,11 @@
           <li data-test-course>
             <LinkTo @route="course" @model={{course}}>
               <FaIcon @icon="square-up-right" />
-              {{course.title}}
+              {{#if this.academicYearCrossesCalendarYearBoundaries}}
+                {{course.title}} ({{course.year}} - {{year-next course.year}})
```

Comment:
I think the title can fall out of this if/else block to avoid needing to be duplicated in both sections.